curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/find-one-or-more-criminals-in-a-hunt.md

```diff
@@ -48,12 +48,12 @@ Your regex should match two criminals (`CC`) in `"CC"`
 assert('CC'.match(reCriminals) && 'CC'.match(reCriminals)[0] == 'CC');
 ```
 
-Your regex should match three criminals (`CCC`) in `"P1P5P4CCCP2P6P3"`
+Your regex should match three criminals (`CCC`) in `"P1P5P4CCCcP2P6P3"`
```

Comment:
We are no longer putting quotes around inline code block values or inside code blocks for strings.  Instead, we want to reword the sentence to something like:
```suggestion
Your regex should match three criminals (`CCC`) in the string `P1P5P4CCCcP2P6P3`.
```

The word "text" could be used instead of "string".

@nhcarrigan Already went through the entire Responsive Web Design (see PR #40838) and removed changed such instances.  The same will need to be done for the other certification sections.